### PR TITLE
Add configs for Karnataka DC

### PR DESCRIPTION
**Pull request by n-h-diaz**
* updates some scripts
* fixes some typos
(will follow up with more documentation edits)

### CODE REVIEW

File: deploy/git/mixer_hash.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+b2ca773
```
Comment (shifucun) [inline deploy/git/mixer_hash.txt:1]: no need to check in this file

File: deploy/git/website_hash.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+90aaec5
```
Comment (shifucun) [inline deploy/git/website_hash.txt:1]: no need to check in this file

File: deploy/gke/karnataka.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
```
Comment (shifucun) [inline deploy/gke/karnataka.yaml:1]: 2022

File: deploy/overlays/karnataka/karnataka.yaml [hunk 4]
```diff
@@ -0,0 +1,658 @@
+apiVersion: v1
```
Comment (shifucun) [inline deploy/overlays/karnataka/karnataka.yaml:1]: no need to check in this file.

File: deploy/overlays/karnataka/kustomization.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Google LLC
```
Comment (shifucun) [inline deploy/overlays/karnataka/kustomization.yaml:1]: 2022

File: deploy/overlays/karnataka/kustomization.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+nameSuffix: -private
```
Comment (shifucun) [inline deploy/overlays/karnataka/kustomization.yaml:19]: change suffix to karnataka

File: deploy/overlays/karnataka/kustomization.yaml [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+patchesStrategicMerge:
+- |-
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: website-app
+  spec:
+    replicas: 9
+images:
```
Comment (shifucun) [inline deploy/overlays/karnataka/kustomization.yaml:45]: This is an updated version of the file. Can you copy it again from other folder, edit it and commit?

I already have a fix that do not update this file but this PR has not sync pass that..

File: deploy/git/mixer_hash.txt (hunk 1 above)
Comment (n-h-diaz) [reply deploy/git/mixer_hash.txt]: done

File: deploy/git/website_hash.txt (hunk 2 above)
Comment (n-h-diaz) [reply deploy/git/website_hash.txt]: done

File: deploy/gke/karnataka.yaml (hunk 3 above)
Comment (n-h-diaz) [reply deploy/gke/karnataka.yaml]: done

File: deploy/overlays/karnataka/karnataka.yaml (hunk 4 above)
Comment (n-h-diaz) [reply deploy/overlays/karnataka/karnataka.yaml]: done

File: deploy/overlays/karnataka/kustomization.yaml (hunk 5 above)
Comment (n-h-diaz) [reply deploy/overlays/karnataka/kustomization.yaml]: done

File: deploy/overlays/karnataka/kustomization.yaml (hunk 6 above)
Comment (n-h-diaz) [reply deploy/overlays/karnataka/kustomization.yaml]: done

File: deploy/overlays/karnataka/kustomization.yaml (hunk 7 above)
Comment (n-h-diaz) [reply deploy/overlays/karnataka/kustomization.yaml]: done

**Comment by n-h-diaz** [comment]
thanks for the review!

**Review by shifucun — APPROVED** [review]

**Pull request merged**